.idea/workspace.xml

```diff
@@ -1,12 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Into your commit got `PHPStorm` file. 
Please, ignore ALL folder `.idea` in `.gitignore` file.